### PR TITLE
7466 add fields to basic project

### PR DESCRIPTION
**Pull request by emitche**
- Add summary and details fields to Basic Project form
- Use same form partial for Loans and Basic Projects

### CODE REVIEW

File: spec/models/basic_project_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe BasicProject, type: :model do
```
Comment (emitche) [inline spec/models/basic_project_spec.rb]: Do I need to add any additional tests for BasicProject?

File: app/controllers/admin/basic_projects_controller.rb [hunk 2]
```diff
@@ -95,7 +95,7 @@ def prep_form_vars
   end
 
   def basic_project_params
-    params.require(:basic_project).permit(:division_id, :length_months, :name, :primary_agent_id,
-      :secondary_agent_id, :signing_date, :status_value)
+    params.require(:basic_project).permit([:division_id, :length_months, :name, :primary_agent_id,
+      :secondary_agent_id, :signing_date, :status_value] + translation_params(:summary, :details))
```
Comment (hooverlunch) [inline app/controllers/admin/basic_projects_controller.rb]: Nice modification here.

File: app/views/admin/projects/_translatable.html.slim [hunk 3]
```diff
@@ -17,5 +17,7 @@
 
 javascript:
   $(function() {
-    new MS.Views.TranslationsView({el: $('[data-content-translatable="loan"]')});
+    new MS.Views.TranslationsView({
+      el: $("[data-content-translatable='#{project.class.name.underscore}']")
```
Comment (hooverlunch) [inline app/views/admin/projects/_translatable.html.slim]: Very nice code reuse!

File: spec/models/basic_project_spec.rb (hunk 1 above)
Comment (hooverlunch) [reply spec/models/basic_project_spec.rb]: No this is fine, no major logic additions in this issue. Nice to have a basic projects spec file now!

**Comment by linesarefuzzy** [comment]
Looks good. We may want to add `simple_format` to the details display on both loans and basic projects.

**Comment by emitche** [comment]
@linesarefuzzy Interesting. That does look useful.

**Pull request merged**